README.rst

```diff
@@ -1 +1,5 @@
 This is the Artefactual Storage Service for Archivematica.
+
+Start worker:
+
+celery -A storage_service worker -l info
```

Comment:
Should https://github.com/artefactual-labs/ansible-role-archivematica-src be updated to start celery too?